### ISSUE TITLE
Peanut Core collection data into CSV format

### ISSUE THREAD

**Issue by sdash-github** [legumeinfo/ArachisPheno#1]
Make a matrix format csv file from my summary spreadsheet of Roshan's trait data.  Then, Sven can check its suitability for loading and other related issues.

**Comment by sdash-github** [comment]
###  ISSUE: Replicate data or Summary data   (TODO Ask?)

AraPheno formats allow for loading data from replicates.  Seems they encourage replicate data instead of just summary data per accession per trait. This being the background:
--  Roshan's data has replicate data available along with average of replicates where available.
--  Hence the question, which data should be loaded into ArachisPheno??

Ref to FAQ on replicates in AraPheno.

### Ask:
SC, RK, EC, PO, ADF

**Comment by sdash-github** [comment]
####  Opinion for long term purpose: load replicate data.
- We should load replicate data in ArachisPheno.  Eventually it is meant for the research community and we should provide them the curated raw data than summarized (avg.) data as it will be more useful statistically for alternate analysis or downstream analysis by another researcher in future.
- If we end of serving it through DataStore, we have the option of providing our summarized data along with the replicate data.

**Comment by adf-ncgr** [comment]
agreed that loading replicates into ArachisPheno is the right thing to do. I think there are charts that allow users to see variability among these measures, for example.

**Comment by piotyama** [comment]
I agree too. I would want to have access to the raw data as a user rather than just averages.  
@sdash-github, might I suggest you wait for a conference call we are scheduled to have with Greg and Noelle about this data? From Noelle's emails, it doesn't look like any of us have the complete dataset from FL 2013/2015 core phenotyping project.

**Comment by sdash-github** [comment]
Hi Sudhansu,
I have another spreadsheet, manually curated phenotype data with replicate information (you might already have it).  Let me know if you need that file, I can send it to you.
Roshan
> Great. I would have eventually looked or asked for it.

**Comment by adf-ncgr** [comment]
> I agree too. I would want to have access to the raw data as a user rather than just averages.
> @sdash-github, might I suggest you wait for a conference call we are scheduled to have with Greg and Noelle about this data? From Noelle's emails, it doesn't look like any of us have the complete dataset from FL 2013/2015 core phenotyping project.

@piotyama thanks for the heads-up on this! we will likely just plow ahead in prototype mode with the data we have since we're trying to get the application into shape before funds expire. But, we won't invest too much in refining the data if as you say what we have now is only partial. thanks again

**Comment by sdash-github** [comment]
On 2020/3/28 11:47 AM, Roshan Kulkarni wrote:
> Hi Sudhansu,
phenotype_data_V1.xls contains the replicate data.
Thanks,
Roshan
